source/apiVolontaria/apiVolontaria/factories.py

```diff
@@ -10,7 +10,7 @@ class Meta:
 
     first_name = factory.Faker('first_name')
     last_name = factory.Faker('last_name')
-    username = 'John'
+    username = factory.Sequence('John{0}'.format)
```

Comment:
From Factory_boy docs:
```
 email = factory.Sequence(lambda n: 'person{0}@example.com'.format(n))
```
Does using a lambda make the code more readable?

Reply:
I'm not a big user of `lambda`. It's a kind of anonymous function you can use to return result of function directly as a parameter. But here it's the same result in more verbose with a usage of `lambda` who can just be scary to read for a new developer.

Reply:
Good for me!